src/overlays/actors/ovl_En_Ma3/z_en_ma3.c

```diff
@@ -209,7 +199,7 @@ u16 func_80AA2AA0(GlobalContext *globalCtx, Actor *this) {
             break;
```

Comment:
Have you tried removing all of these cases? Seems a bit unneeded.

Reply:
It looks like I can get away with only doing cases 0 and 9, but I thought it was more explicit and clear to list them all.